DayrSuryan/DSEthiop7.xml

```diff
@@ -345,14 +418,22 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                <history>
                   <origin>
-                     <origDate notBefore="1927" notAfter="1928"/>
-                     The manuscript could be copied
-                     by <persName ref="PRS11959Pawlos" role="scribe"/>,
-                     in <placeName ref="LOC2804Egypt">Egypt</placeName> or in <placeName ref="LOC2991Eritre">Eritrea</placeName>.
-                     A brief note identifies <persName ref="PRS12557Barha">ḫalaqā Barha</persName> as the owner.
-                   </origin>
-                  <provenance/>
+                     <origDate notBefore="1900" notAfter="1949">First half of the twentieth century</origDate>
+                     The  <ref target="#coloph1">colophon</ref> identifies  <persName ref="PRS12557Barha">ḫalaqā Barha</persName> as
+                     the owner. According to a note added probably later, in a secondary hand,
+                     the scribe was <persName ref="PRS11959Pawlos" role="scribe"/> who copied the book in
+                     <placeName ref="LOC2991Eritre">Eritrea</placeName> or <placeName ref="LOC2804Egypt">Egypt</placeName>
+                     and owned it.</origin>
```

Comment:
Now I see that we can surpress Egypt as very minor chance, because the text  on fol. 52r directly adresses the community of Dabra Bizan. Smth like "... copied in Eritrea, in the milieu of Dabra Bizan"?